code/core/src/com/ece454/gotl/GotlGame.java

```diff
@@ -14,32 +15,40 @@
 import com.badlogic.gdx.physics.box2d.Box2DDebugRenderer;
 import com.badlogic.gdx.physics.box2d.World;
 
+import handlers.AssetHandler;
 import handlers.GameStateManager;
 import handlers.WorldManager;
+import screens.LevelCompleteScreen;
 import states.PlayState;
 
-public class Game extends ApplicationAdapter {
+public class GotlGame extends Game {
 	public static final float PIXEL_PER_METER = 32f;
 
 	public static final float SCALE = 2.0f;
 	public static final float TIME_STEP = 1 / 60f;
 	public static final int VELOCITY_ITERATIONS = 6;
 	public static final int POSITION_ITERATIONS = 2;
-	private SpriteBatch batch;
+	public SpriteBatch batch;
 	private GameStateManager gsm;
 	private Box2DDebugRenderer box2DDebugRenderer;
+	public AssetHandler assetHandler;
 
 	@Override
 	public void create () {
 		batch = new SpriteBatch();
 		gsm = new GameStateManager(this);
-		gsm.push(new PlayState(gsm));
+		assetHandler = new AssetHandler();
+		assetHandler.loadAssets();
+		PlayState playScreen = new PlayState(gsm);
+		gsm.push(playScreen);
```

Comment:
No need for gsm if we're using com.badlogic.gdx.Game. IMO we should keep the game state manager to actually implement some architecture ourselves instead of letting the engine abstract everything away.

Reply:
discussed offline, will look into refactoring it using game state manager